app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/edit/nutrition/ProductEditNutritionFactsFragment.kt

```diff
@@ -712,17 +711,25 @@ class ProductEditNutritionFactsFragment : ProductEditFragment() {
         }
 
     private fun displayAddNutrientDialog() {
-        val nutrients = resources.getStringArray(R.array.nutrients_array)
+        val nutrientsDefUnits = resources.getStringArray(R.array.nutrients_array).zip(PARAMS_OTHER_NUTRIENTS_DEFAULT_UNITS.keys)
+        val filteredNutrients = resources.getStringArray(R.array.nutrients_array)
             .filterIndexed { index, _ -> index !in usedNutrientsIndexes }
-            .sortedWith(Collator.getInstance(Locale.getDefault()))
+            .sortedBy { it.lowercase() }
```

Comment:
Why did you remove this? By using `sortedBy { it.lowercase() }` we're not considering all Unicode languages.

Reply:
Sorry, was a part of one of a previous implementation idea that we forgot to clean up. But now it's back